profiler/hotapp/main.go

```diff
@@ -75,6 +75,18 @@ func allocImpl() {
 	}
 }
 
+// Simulates a function which allocates a lot of memory, but does not hold on
```

Comment:
Comments should start with the thing being described. For example,

```
// allocMany simulates a function...
```

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
Done!